### DISCUSSION TITLE
two background_tasks , Is it the second background_tasks must wait for the first background_tasks to execute before the second Add_task

### DISCUSSION THREAD

**Discussion by jaffe-fly** [fastapi/fastapi#9359] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import uvicorn
from fastapi import FastAPI, BackgroundTasks
import time

app = FastAPI()


async def test_func(s: str) -> bool:
    time.sleep(5)
    print(f"~~~ test_func: {s}")
    return True


@app.get("/test/v1")
def get_test(background_tasks: BackgroundTasks) -> dict:
    background_tasks.add_task(test_func, "one") # Only this task's output will be logged
    background_tasks.add_task(test_func, "two")
    return {"message": "test"}



if __name__ == '__main__':
    uvicorn.run('scratch_1:app', host='0.0.0.0', port=5000, workers=1, reload=True)
```

### Description

Is the second background_tasks must wait for the first background_tasks to execute,
so how to start the two background_tasks at the same time?

### Operating System

Linux

### FastAPI Version

0.78.0

### Python Version

3.9

**Comment by YuriiMotov** [comment]
Couldn't reproduce this. Works well
